src/atlas.nim

```diff
@@ -72,6 +75,7 @@ Options:
   --list                list all available and installed versions
   --version             show the version
   --help                show this help
+  --global              show this help
```

Comment:
I doubt it..

Reply:
lol, good point. I'm unsure about the Global bit, should I remove it? I think Nimble already handles that case alright.

Reply:
Well Nimble uses a different dependency resolution algorithm and we don't want to depend on it for good.